client/app/components/Input/InputSelect.jsx

```diff
@@ -45,6 +45,7 @@ export function InputSelect({
         aria-label={label || ariaLabel}
         value={value}
         onChange={toggleValue}
+        data-testid="inputSelect"
```

Comment:
So putting a `data-testid` is not necessary here because we can query this element using it's `aria-label` instead or even by the fact that it's a native `<select>`.

Reply:
Since you're not working on migrating this test file, I would not include this in your PR.